### PR TITLE
Feature: updated age example with massaExport

### PR DESCRIPTION
**Pull request by 0xSwapFeeder**
Solves #83

### CODE REVIEW

File: age/smart-contract/asconfig.json [hunk 1]
```diff
@@ -3,15 +3,17 @@
         "debug": {
             "outFile": "build/main.wasm",
             "sourceMap": true,
-            "debug": true
+            "debug": true,
+            "transformer": ["@massalabs/as-transformer"]
```
Comment (julienbrs) [inline age/smart-contract/asconfig.json:7]:
```suggestion
            "transform": ["@massalabs/as-transformer"]
```

File: age/smart-contract/asconfig.json [hunk 2]
```diff
@@ -3,15 +3,17 @@
...
         },
         "release": {
             "outFile": "build/main.wasm",
             "sourceMap": true,
             "optimizeLevel": 3,
             "shrinkLevel": 0,
             "converge": false,
-            "noAssert": false
+            "noAssert": false,
+            "transformer": ["@massalabs/as-transformer"]
```
Comment (julienbrs) [inline age/smart-contract/asconfig.json:16]:
```suggestion
            "transform": ["@massalabs/as-transformer"]
```

**Comment by julienbrs** [comment]
lgtm, could you put in package.json the latest version of transformer that works please?

**Comment by gregLibert** [comment]
The CI is not wokking. Do you know if this code should target the buildnet or the testnet ?

**Review by gregLibert — APPROVED** [review]
We need to fix the CI tho

**Comment by 0xSwapFeeder** [comment]
> The CI is not wokking. Do you know if this code should target the buildnet or the testnet ?

I'm not aware of compatibilities issues with tooling, my guess would be the testnet since it always worked on it however I don't know if the tooling works with the current testnet.

**Pull request closed without merging**